packages/core/src/util/config.ts

```diff
@@ -22,6 +22,7 @@ import {
   SHADOW_OPACITY,
   SHADOWCOLOR,
 } from './Constants';
+import { copyProperties } from './cloneUtils';
```

Comment:
@coderabbitai what do you think of the updated implementation of the function?

Reply:
This function explicitly mentions both in its name and in its JSDoc that it is performing a shallow copy on purpose.
It will be use internally only to copy objects that are know to only have primitive properties.
That's why I am not planning to implement deep copy.